spec/core_functions/color/hsl/one_arg/relative_color.hrx

```diff
@@ -0,0 +1,125 @@
+<===> static/alpha/input.scss
+a {b: hsl(from #aaa h s l / 25%)}
+
+<===> static/alpha/output.css
+a {
+  b: hsl(from #aaa h s l/25%);
+}
+
+<===>
+================================================================================
+<===> static/no_alpha/input.scss
+a {b: hsl(from #aaa h s l)}
+
+<===> static/no_alpha/output.css
+a {
+  b: hsl(from #aaa h s l);
+}
+
+<===>
+================================================================================
+<===> calc/alpha/input.scss
+a {b: hsl(from #aaa calc(h + 180deg) s l / 25%)}
+
+<===> calc/alpha/output.css
+a {
+  b: hsl(from #aaa calc(h + 180deg) s l/25%);
+}
+
+<===>
+================================================================================
+<===> calc/no_alpha/input.scss
+a {b: hsl(from #aaa calc(h + 180deg) s l)}
+
+<===> calc/no_alpha/output.css
+a {
+  b: hsl(from #aaa calc(h + 180deg) s l);
+}
+
+<===>
+================================================================================
+<===> var/alpha/input.scss
+a {b: hsl(from var(--c) h s l / 25%)}
+
+<===> var/alpha/output.css
+a {
+  b: hsl(from var(--c) h s l/25%);
+}
+
+<===>
+================================================================================
+<===> var/no_alpha/input.scss
+a {b: hsl(from var(--c) h s l)}
+<===> var/no_alpha/output.css
```

Comment:
Nit: [end each file with a newline](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-end-each-file-with-a-newline)

Reply:
looks like the spec linter does not enforce this right now.

Reply:
The spec linter is unfortunately not as comprehensive as we would like it to be, but it's hard to allocate time to it instead of more pressing concerns.